run.sh

```diff
@@ -13,6 +13,44 @@ if [ -z "$SUMO_ACCESS_ID" ] || [ -z "$SUMO_ACCESS_KEY" ]; then
 	exit 1
 fi
 
+# Support using env as replacement within sources.
+# Gather all template files
+declare -a TEMPLATE_FILES
+if [ -r "${SUMO_SOURCES_JSON}.tmpl" ]; then
+    TEMPLATE_FILES+=("${SUMO_SOURCES_JSON}.tmpl")
+fi
+if [ -r "${SUMO_SYNC_SOURCES}.tmpl" ]; then
```

Comment:
I don't think line 22-27 is needed here, `SUMO_SYNC_SOURCES` is a boolean that specifies whether collector should continuously monitor `SUMO_SOURCES_JSON` for automatic configuration update